sort/sort.go

```diff
@@ -25,7 +25,13 @@ func MultithreadedSortComparators(comparators Comparators) Comparators {
 	copy(toBeSorted, comparators)
 
 	var wg sync.WaitGroup
-	chunks := chunk(toBeSorted, int64(runtime.NumCPU()))
+
+	numCPU := int64(runtime.NumCPU())
+	if numCPU%2 == 1 { // single core machine
+		numCPU++
+	}
```

Comment:
Yeah, and AMD at least went through a phase where they sold triple-core processors. I actually have one at home in my desktop; fourth core unlocked and overclocked of course :smile: